modules/packages/cargo/parser.go

```diff
@@ -137,14 +137,15 @@ func parsePackage(r io.Reader) (*Package, error) {
 	dependencies := make([]*Dependency, 0, len(meta.Deps))
 	for _, dep := range meta.Deps {
 		dependencies = append(dependencies, &Dependency{
-			Name:            dep.Name,
+			Name:            dep.ExplicitNameInToml,
```

Comment:
This should probably default to `dep.Name` if `dep.ExplicitNameInToml` is not set